tests/index_test.js

```diff
@@ -9,12 +9,205 @@
 'use strict';
 
 const tape = require('tape'),
-  validator = require('../index');
+  validator = require('../index'),
+  fs = require('fs');
```

Comment:
Separate native modules from other modules by listing them in a separate const that appears as first